common/pom.xml

```diff
@@ -46,17 +46,22 @@
                         <configuration>
                             <artifacts>
                                 <artifact>
-                                    <file>${project.basedir}/src/main/resources/docker/docker.bom</file>
+                                    <file>catalog/docker/docker.bom</file>
```

Comment:
Not sure if we still need `${project.basedir}` I know this was needed when using maven with `-f`